docs/src/main/asciidoc/security-service-accounts.adoc

```diff
@@ -20,4 +20,56 @@ For earlier versions, it needs to be specified as an environment variable to the
 === Service Account
 
 For distributions of Kubernetes that support more fine-grained role-based access within the cluster, you need to make sure a pod that runs with `spring-cloud-kubernetes` has access to the Kubernetes API.
-For any service accounts you assign to a deployment or pod, you need to make sure they have the correct roles. For example, you can add `cluster-reader` permissions to your `default` service account, depending on the project you're in.
+For any service accounts you assign to a deployment or pod, you need to make sure they have the correct roles.
+
+Depending on the requirements, you'll need `get`, `list` and `watch` permission on the following resources:
+
+.Kubernetes Resource Permissions
+|===
+|Dependency | Resources
+
+
+|spring-cloud-starter-kubernetes
+|pods, services, endpoints
+
+|spring-cloud-starter-kubernetes-config
+|configmaps, secrets
+
+|spring-cloud-starter-kubernetes-ribbon
+|pods, services, endpoints
+|===
+
+For development purposes, you can add `cluster-reader` permissions to your `default` service account. On a productive system you'll want to provide more granular permissions.
```

Comment:
`productive` -> `production`

Reply:
`you'll want` would probably better as `you'll likely want`

Reply:
+1 thanks!

Reply:
YW!